lib/vbox/machine.go

```diff
@@ -184,11 +183,11 @@ func CheckUpdate(typeFlag string) (string, bool) {
 	return typeFlag, true
 }
 
-// Stops running machines
+// StopMachines stops running machines
 func StopMachines() error {
 	var (
 		answer string
-		prompt bool = true
+		prompt = true
```

Comment:
__[gometalinter]__
* [ineffassign] ineffectual assignment to prompt

<!-- Comment by [SideCI](https://sideci.com) -->